app/graphql/types/query_type.rb

```diff
@@ -37,7 +37,7 @@ class QueryType < Types::BaseObject
     field :memberships, resolver: Resolvers::MembershipsResolver
     field :mrrs, resolver: Resolvers::Analytics::MrrsResolver
     field :organization, resolver: Resolvers::OrganizationResolver
-    field :outstanding_invoices, resolver: Resolvers::Analytics::OutstandingInvoicesResolver
+    field :finalized_invoices, resolver: Resolvers::Analytics::FinalizedInvoicesResolver
```

Comment:
GraphQL/OrderedFields: Fields should be sorted in an alphabetical order within their section. Field `finalized_invoices` should appear before `organization`.